docs/guide.rst

```diff
@@ -1921,15 +1921,14 @@ Dependabot
 
 Dependabot_ creates pull requests with automated dependency updates.
 
-Follow these steps to set up Dependabot for your repository:
+Enable `Security Updates` from the `Security` tab on the repository under `Dependabot alerts` > `Dependabot Security Updates`.
 
-1. Sign up at Dependabot_.
-2. Install their GitHub app.
+More on `official documentation`_ for GitHub Dependabot.
```

Comment:
I would change the phrasing a little. Also seems better to use an anonymous link, because `official documentation` is a bit ambiguous in the User Guide as a whole, and this is the only reference anyway.

```suggestion
Please refer to the `official documentation`__ for more details.

__ https://docs.github.com/en/github/administering-a-repository/keeping-your-dependencies-updated-automatically
```